web/models.py

```diff
@@ -7,7 +7,7 @@
 db_name = config('SQL_DB')
 
 app = Flask(__name__)
-app.config['SQLALCHEMY_DATABASE_URI'] = 'postgresql://{}:{}@db/{}'.format(db_user, db_password, db_name)
+app.config['SQLALCHEMY_DATABASE_URI'] = 'postgresql://{}:{}@localhost/{}'.format(db_user, db_password, db_name)
```

Comment:
Return to db for docker

```suggestion
app.config['SQLALCHEMY_DATABASE_URI'] = 'postgresql://{}:{}@db/{}'.format(db_user, db_password, db_name)
```